src/components/generic/Banner/Banner.jsx

```diff
@@ -25,9 +31,11 @@ Banner.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
   style: PropTypes.shape(),
+  type: PropTypes.string,
```

Comment:
>   // You can ensure that your prop is limited to specific values by treating
  // it as an enum.
  optionalEnum: PropTypes.oneOf(['News', 'Photos']),

https://reactjs.org/docs/typechecking-with-proptypes.html